dev-cpp/gstreamermm/gstreamermm-1.10.0.recipe

```diff
@@ -0,0 +1,88 @@
+SUMMARY="C++ bindings for GStreamer"
+DESCRIPTION="GStreamerMM provides C++ bindings for the GStreamer streaming multimedia \
+library.  With gstreamermm it is possible to develop applications that work with multimedia in C++.
+
+gstreamermm is developed over glibmm, libsigc++ and libxml++ and the functionalities they \
+provide.  This means that, among other things, referencing and unreferencing of GObjects is \
+handled automatically via glibmm's automatic pointer class, Glib::RefPtr, and libsigc++'s \
+slots are used for callbacks and signals."
+HOMEPAGE="https://gitlab.gnome.org/GNOME/gstreamermm"
+COPYRIGHT="2009-2017 GStreamerMM team"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+branch="`echo "$portVersion" | cut -b1-4`"
+SOURCE_URI="https://download.gnome.org/sources/gstreamermm/$branch/gstreamermm-$portVersion.tar.xz"
+CHECKSUM_SHA256="be58fe9ef7d7e392568ec85e80a84f4730adbf91fb0355ff7d7c616675ea8d60"
+PATCHES="gstreamermm-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURE="x86"
+
+libVersion="1.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	gstreamermm$secondaryArchSuffix = $portVersion
+	lib:libgstreamermm_1.0 = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	gst_plugins_base$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libglibmm$secondaryArchSuffix
+	lib:libgstreamer_1.0$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libsigc_2.0$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	gstreamermm${secondaryArchSuffix}_devel = $portVersion
+	devel:libgstreamermm_1.0$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	gstreamermm$secondaryArchSuffix == $portVersion base
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libglibmm$secondaryArchSuffix
+	devel:libgstreamer_1.0$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libglibmm$secondaryArchSuffix
+	devel:libgstreamer_1.0$secondaryArchSuffix
+	gst_plugins_base${secondaryArchSuffix}_devel
```

Comment:
Move up right after haiku_devel (same as in REQUIRES)

Reply:
And done :D